### PR TITLE
Updates readme.

### PR DESCRIPTION
**Pull request by joeldudley**
### Why is this change needed?

There were several todos in the readme that needed to be addressed.

### What changes were made as part of this PR:

Functional.

- Address todos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,6 @@ The typical blockchain node runs multiple services in a single process. For exam
 Obscuro uses Trusted Execution Environments (TEE), like Intel SGX, to execute transactions in a confidential environment, which means we diverge from the typical architecture.
```
Comment (tudor-malene) [inline README.md:17]: can't comment on the diagram. so will add here

On the diagram, can we represent somehow which connections are encrypted and which not. 
that should be very useful for readers.
For example on the rpc from wallet to we- unencrypted, and from we-host - encrypted

File: README.md [hunk 2]
```diff
@@ -180,15 +179,13 @@ building blocks of the Enclave*
 
 ### III. The Wallet Extension
 
-The missing link to achieving fully private transactions, while allowing end users to continue using their favourite wallets
-(like MetaMask) is a very thin component that has to be installed on the user machine.
-This component is responsible for the encryption/decryption of the traffic originating from an Obscuro node. It does that 
-by generating Viewing Keys behind the scenes.
+The missing link to achieving fully private transactions while allowing end-users to continue using their favourite 
+wallets (like MetaMask). This is a very thin component that is responsible for the encrypting and decrypting traffic
```
Comment (tudor-malene) [inline README.md:183]: "for the encryption and decryption .." or "for encrypting or decrypting"

File: README.md [hunk 3]
```diff
@@ -180,15 +179,13 @@ building blocks of the Enclave*
...
-This component is responsible for the encryption/decryption of the traffic originating from an Obscuro node. It does that 
-by generating Viewing Keys behind the scenes.
+The missing link to achieving fully private transactions while allowing end-users to continue using their favourite 
+wallets (like MetaMask). This is a very thin component that is responsible for the encrypting and decrypting traffic 
+between the Obscuro node and its clients.
 
-[//]: # (TODO - Joel: want to add anything here)
-
-See: [tools/walletextension](tools/walletextension)
+For security reasons, this component must be run locally by the user.
```
Comment (tudor-malene) [inline README.md:186]: you should probably mention here that it's a standalone program that has an rpc interface similar to an ethereum node, and translates and encrypts stuff and submits it to the obscuro rpc. 
And , responses ...
And viewing keys...

**Review by tudor-malene — COMMENTED** [review]
few comments

File: README.md (hunk 2 above)
Comment (joeldudley) [reply README.md:183]: Fixed.

File: README.md (hunk 3 above)
Comment (joeldudley) [reply README.md:186]: Yeah, sounds good :)

File: README.md (hunk 1 above)
Comment (joeldudley) [reply README.md:17]: Fixed.

**Review by tudor-malene — APPROVED** [review]
lgtm

**Pull request merged**